azerothcore/azerothcore-wotlk
# fix(DB/Quest_template): Correct handin location for Twisted Hatred

## Changes Proposed:
Fixes the displayed handin location for [Twisted Hatred](https://tbc.wowhead.com/quest=932/twisted-hatred). It was sending players to Lake Al'Ameth instead of Dolanaar.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7924
- No corresponding CC issue as this issue was raised in a comment elsewhere. (https://github.com/chromiecraft/chromiecraft/issues/1696#issuecomment-919603494) 

## SOURCE:

We can't definitely say if incorrect quest handin locations are not Blizz-like, but basic consistency would argue that they are not.

## Tests Performed:

- Ran SQL, no errors/warnings, 1 row affected as expected.
- Tested in-game (had to clear cache first before it showed up):

![WoWScrnShot_091821_162717](https://user-images.githubusercontent.com/81782124/133879548-2cb6978a-b4f4-4472-a2bf-633b10b7f7b8.jpg)
![WoWScrnShot_091821_162721](https://user-images.githubusercontent.com/81782124/133879551-ba73e5eb-a610-47bb-8347-bd804bcd1a34.jpg)

## How to Test the Changes:


1. .q add 932
2. .go c 49850
3. Separate head from demon (the demon may not cooperate with this bit.)
4. Check quest tracker

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
